libsrc/.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
I don't see ignoring Git all kinds of temporary files of all possible editors like '.bak' etc. If you don't like to see them make your editor not create them.